src/ape/cli/options.py

```diff
@@ -145,6 +145,24 @@ def skip_confirmation_option(help=""):
     )
 
 
+def upgrade_confirmation_option(help=""):
+    """
+    A ``click.option`` for upgrade plugin confirmation (``--upgrade``).
+
+    Args:
+        help (str): CLI option help text. Defaults to ``""``.
+    """
+
+    return click.option(
+        "-U",
+        "--upgrade",
+        "upgrade_confirmation",
```

Comment:
We don't need this part - `upgrade_confirmation`.
If you exclude this, you can call the parameter `upgrade` inside the CLI command method